MSAL/test/unit/MSALAcquireTokenTests.m

```diff
@@ -53,6 +69,10 @@ @implementation MSALAcquireTokenTests
 - (void)setUp
 {
     [super setUp];
+    
+#if TARGET_OS_IPHONE
+    [MSIDKeychainTokenCache reset];
```

Comment:
can we use cache clearAllItems here instead, so it's universal for both iOS and macOS?

Reply:
done